2023-04-24 06:29:01,008 INFO    Thread-44 :5492 [wandb_setup.py:_flush():68] Configure stats pid to 5492
2023-04-24 06:29:01,008 INFO    Thread-44 :5492 [wandb_setup.py:_flush():68] Loading settings from C:\Users\palas\.config\wandb\settings
2023-04-24 06:29:01,008 INFO    Thread-44 :5492 [wandb_setup.py:_flush():68] Loading settings from c:\Users\palas\Desktop\IOT\cs6910_assignment3\wandb\settings
2023-04-24 06:29:01,008 INFO    Thread-44 :5492 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True', 'project': 'rnn_runs', 'entity': 'nihil666', 'root_dir': 'c:\\Users\\palas\\Desktop\\IOT\\cs6910_assignment3', 'run_id': 'f3mfvkc2', 'sweep_param_path': 'c:\\Users\\palas\\Desktop\\IOT\\cs6910_assignment3\\wandb\\sweep-u1n7ugt4\\config-f3mfvkc2.yaml', 'sweep_id': 'u1n7ugt4'}
2023-04-24 06:29:01,008 INFO    Thread-44 :5492 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-04-24 06:29:01,008 INFO    Thread-44 :5492 [wandb_init.py:_log_setup():492] Logging user logs to c:\Users\palas\Desktop\IOT\cs6910_assignment3\wandb\run-20230424_062901-f3mfvkc2\logs\debug.log
2023-04-24 06:29:01,009 INFO    Thread-44 :5492 [wandb_init.py:_log_setup():493] Logging internal logs to c:\Users\palas\Desktop\IOT\cs6910_assignment3\wandb\run-20230424_062901-f3mfvkc2\logs\debug-internal.log
2023-04-24 06:29:01,009 INFO    Thread-44 :5492 [wandb_init.py:_jupyter_setup():438] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001AD97CDF6A0>
2023-04-24 06:29:01,009 INFO    Thread-44 :5492 [wandb_init.py:init():532] calling init triggers
2023-04-24 06:29:01,009 INFO    Thread-44 :5492 [wandb_init.py:init():538] wandb.init called with sweep_config: {'bidirectional': 0, 'cell_type': 'GRU', 'dropout': 0.2, 'hidden_layer_size': 64, 'input_embedding_size': 128, 'num_decoder_layers': 1, 'num_encoder_layers': 2, 'teacher_forcing': 0.5}
config: {}
2023-04-24 06:29:01,009 INFO    Thread-44 :5492 [wandb_init.py:init():588] starting backend
2023-04-24 06:29:01,009 INFO    Thread-44 :5492 [wandb_init.py:init():592] setting up manager
2023-04-24 06:29:01,015 INFO    Thread-44 :5492 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn, using: spawn
2023-04-24 06:29:01,021 INFO    Thread-44 :5492 [wandb_init.py:init():599] backend started and connected
2023-04-24 06:29:01,033 INFO    Thread-44 :5492 [wandb_run.py:_config_callback():1250] config_cb None None {'bidirectional': 0, 'cell_type': 'GRU', 'dropout': 0.2, 'hidden_layer_size': 64, 'input_embedding_size': 128, 'num_decoder_layers': 1, 'num_encoder_layers': 2, 'teacher_forcing': 0.5}
2023-04-24 06:29:01,034 INFO    Thread-44 :5492 [wandb_run.py:_label_probe_notebook():1203] probe notebook
2023-04-24 06:29:01,034 INFO    Thread-44 :5492 [wandb_run.py:_label_probe_notebook():1213] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-04-24 06:29:01,034 INFO    Thread-44 :5492 [wandb_init.py:init():687] updated telemetry
2023-04-24 06:29:01,076 INFO    Thread-44 :5492 [wandb_init.py:init():727] communicating run to backend with 60.0 second timeout
2023-04-24 06:29:01,717 INFO    Thread-44 :5492 [wandb_run.py:_on_init():2134] communicating current version
2023-04-24 06:29:01,766 INFO    Thread-44 :5492 [wandb_run.py:_on_init():2143] got version response upgrade_message: "wandb version 0.15.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-04-24 06:29:01,767 INFO    Thread-44 :5492 [wandb_init.py:init():775] starting run threads in backend
2023-04-24 06:29:01,935 INFO    Thread-44 :5492 [wandb_run.py:_console_start():2114] atexit reg
2023-04-24 06:29:01,935 INFO    Thread-44 :5492 [wandb_run.py:_redirect():1969] redirect: SettingsConsole.WRAP_RAW
2023-04-24 06:29:01,935 INFO    Thread-44 :5492 [wandb_run.py:_redirect():2034] Wrapping output streams.
2023-04-24 06:29:01,935 INFO    Thread-44 :5492 [wandb_run.py:_redirect():2059] Redirects installed.
2023-04-24 06:29:01,935 INFO    Thread-44 :5492 [wandb_init.py:init():817] run started, returning control to user process
2023-04-24 07:13:45,188 INFO    Thread-44 :5492 [wandb_run.py:_finish():1854] finishing run nihil666/rnn_runs/f3mfvkc2
2023-04-24 07:13:45,188 INFO    Thread-44 :5492 [jupyter.py:save_history():449] not saving jupyter history
2023-04-24 07:13:45,188 INFO    Thread-44 :5492 [jupyter.py:save_ipynb():379] not saving jupyter notebook
2023-04-24 07:13:45,188 INFO    Thread-44 :5492 [wandb_init.py:_jupyter_teardown():420] cleaning up jupyter logic
2023-04-24 07:13:45,188 INFO    Thread-44 :5492 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2023-04-24 07:13:45,189 INFO    Thread-44 :5492 [wandb_run.py:_restore():2066] restore
2023-04-24 07:13:45,189 INFO    Thread-44 :5492 [wandb_run.py:_restore():2072] restore done
2023-04-24 07:13:49,979 INFO    Thread-44 :5492 [wandb_run.py:_footer_history_summary_info():3442] rendering history
2023-04-24 07:13:49,979 INFO    Thread-44 :5492 [wandb_run.py:_footer_history_summary_info():3474] rendering summary
2023-04-24 07:13:49,984 INFO    Thread-44 :5492 [wandb_run.py:_footer_sync_info():3398] logging synced files
